integrations/servicenow/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,52 @@
+resources:
+  - kind: group
+    selector:
+      query: 'true'
+      path: 'sys_user_group'
```

Comment:
I dont like this 
Its 2 generic

Is their api really that generic 
The routes are not known?

And if it does then you can just use the kind instead